dsgrid/dataset/dataset_schema_handler_standard.py

```diff
@@ -90,6 +98,7 @@ def make_project_dataframe_from_query(self, context: QueryContext, project_confi
 
         convert_time_before_project_mapping = self._convert_time_before_project_mapping()
         if convert_time_before_project_mapping:
+            # There is currently no case that needs model years or value columns.
```

Comment:
Ditto again. I guess another approach could be just to pass them in. Is it confusing because we don't know what order things are mapped in in this case, or can we assume that everything except geography has already been mapped?

Reply:
I was avoiding work - and writing code that cannot be tested. The mapping starts a few lines down (104). Value columns at this point are from the dataset, not the project.